src/index.js

```diff
@@ -34,9 +35,11 @@ const Index = () => {
   };
   return (
     <React.StrictMode>
-      <SearchContext.Provider value={value}>
-        <RouteDefs />
-      </SearchContext.Provider>
+      <ErrorBoundry>
```

Comment:
I would prefer to have the navbar intact while showing errorboundary component. There are two options 
- We can re-create navbar with minimalistic (only the logo) elements and put inside error boundary component
- Create the error boundary at route level instead of root level

Do you agree?

Reply:
@koustov , I think we should not show the navbar when the page goes blank because it's irrelevant.
Error boundary should be at the root level so that we can catch all the errors.